soc/canaan/k230/Kconfig.soc

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Junhui Liu <liujh2818@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_CANAAN_K230
+	bool
+	select SOC_FAMILY_CANAAN
+
+config SOC_CANAAN_K230
+	bool
+	select SOC_SERIES_CANAAN_K230
+
+config SOC_SERIES
+	default "k230" if SOC_SERIES_CANAAN_K230
+
+config SOC
+	default "k230" if SOC_CANAAN_K230
```

Comment:
for the text here, it needs to match the symbol, if you have k230 then you need SOC_SERIES_K230/SOC_K230, if you have canaan_k280 then you need SOC_SERIES_CANAAN_K230/SOC_CANAAN_K230

Reply:
Got it, I'll update it when rebase